diagnostic.md

```diff
@@ -99,6 +124,12 @@ div {
 ```
 
 <!-- your answer starts here -->
+.specialDiv
+
+- color: red;
+- font-size: 16px;
+- font-family: cursive;
+
```

Comment:
color here will be yellow - its class styling is more specific than its parent's cascaded style